public/app/features/dashboard/dashgrid/DashboardPanel.tsx

```diff
@@ -89,9 +89,19 @@ class DashboardPanelWithOutEvents extends PureComponent<Props, State> {
   }
 
   componentDidMount() {
-    this.loadPlugin(this.props.panel.type);
-    this.props.events(`Panel:${this.props.panel.id}`).subscribe(event => {
-      console.log(`New Event received from ${event.origin} received by Panel:${this.props.panel.id}`, event);
+    const { panel } = this.props;
+    this.loadPlugin(panel.type);
+    this.props.subscribeToEvents({
+      filter: event => {
+        if (event.origin === panel.id.toString()) {
+          return false;
+        }
+
+        return event.name === mouseMoveEvent.name;
+      },
+      tap: event => {
+        // console.log('Event', event);
+      },
```

Comment:
Could need some help with naming here. `tap`and `filter` is the rxjs equivalent but maybe not the best names for someone unfamiliar with rxjs.